bitnami/mariadb-galera/README.md

```diff
@@ -532,6 +572,18 @@ $ helm upgrade my-release bitnami/mariadb-galera \
 
 | Note: you need to substitute the placeholders _[ROOT_PASSWORD]_, _[MARIADB_PASSWORD]_ and _[MARIABACKUP_PASSWORD]_ with the values obtained from instructions in the installation notes.
 
+### To 7.0.0
+
+This major release renames several values in this chart and adds missing features, in order to be inline with the rest of assets in the Bitnami charts repository. Also, this release address issues related to cluster initialization and node's restarts.
+
+Affected values:
+
+- `service.port` renamed as `service.ports.mysql`.
+- `service.nodePort` renamed as `service.nodePorts.mysql`.
+- `securityContext` renamed as `podSecurityContext`.
+- `extraInitContainers` renamed as `initContainers`.
+- `prometheusRule.selector` renamed as `prometheusRule.additionalLabels`
+
```

Comment:
Am I supposed to run any special steps to upgrade from `6.x.x`? If so, please document them.

Reply:
No special steps are needed.